src/components/atoms/Burger/index.tsx

```diff
@@ -7,7 +7,6 @@ interface BurgerInterface {
 }
 
 const BurgerIcon: React.FC<BurgerInterface> = (props: BurgerInterface) => {
-    console.log(props.open)
     return (
         <StyleBurger open={props.open} onClick={props.setOpen}>
             <StyleDiv open={props.open} />
```

Comment:
I would rename `StyleDiv` for something easier to understand, for example, `StyleContainer` or whatever the div is.

Reply:
Good point. I renamed it `StyleIconBar ` since they represent the lines/bars of the burger icon.